temboardui/utils.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from time import sleep
+
+logger = logging.getLogger(__name__)
+
+
+def check_sqlalchemy_connectivity(engine):
+    for i in range(10):
+        try:
+            conn = engine.connect()
+            conn.close()
+            break
+        except Exception as e:
+            if i == 9:
+                raise
+            logger.warn("Failed to connect to database: %s", e)
+            logger.info("Retrying in %ss.", i)
+            sleep(i)
```

Comment:
yes. It's a linear backoff. It allows to quickly retry if postgres is there while avoiding tons of logs if postgres is waiting.

The long time can be when you launch temboard and trigger compose up afterward, (you read the doc, open another terminal, etc.)